Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -602,6 +797,11 @@ ResourceLoaderOptions loaderOptions(
 void MediaPlayerPrivateWebM::notifyClientWhenReadyForMoreSamples(uint64_t trackId)
 {
     if (trackId == m_enabledVideoTrackID) {
+        if (m_decompressionSession) {
+            m_decompressionSession->requestMediaDataWhenReady([this, trackId] {
+                didBecomeReadyForMoreSamples(trackId);
+            });
+        }
```

Comment:
Can we guarantee that the player will still be valid when the lambda is called? If not, you should probably use a WeakPtr like you do for the callbacks from the audio renderer and the AVSBDL